test/e2e/setup/setup.go

```diff
@@ -228,8 +229,8 @@ func marshalRuntimeObjectFromYAMLBytes(bytes []byte, obj runtime.Object) error {
 	return json.Unmarshal(jsonBytes, &obj)
 }
 
-func createOrUpdate(obj client.Object) error {
-	if err := e2eutil.TestClient.Create(context.TODO(), obj, &e2eutil.CleanupOptions{}); err != nil {
+func createOrUpdate(obj client.Object, ctx *e2eutil.Context) error {
```

Comment:
[nit] we should keep argument order consistent, ctx has been the first argument everywhere else.